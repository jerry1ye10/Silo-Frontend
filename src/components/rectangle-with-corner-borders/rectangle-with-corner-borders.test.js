import React from 'react';
import renderer from 'react-test-renderer';

import {RectangleWithCornerBorders} from '.';

describe('<RectangleWithCornerBorders />', () => {
  it('snapshot', () => {
    const component = renderer.create(<RectangleWithCornerBorders />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
