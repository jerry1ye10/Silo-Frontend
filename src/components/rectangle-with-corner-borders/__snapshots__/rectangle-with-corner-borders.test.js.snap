// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RectangleWithCornerBorders /> snapshot 1`] = `
<View
  style={
    Object {
      "aspectRatio": 1,
      "backgroundColor": "transparent",
      "width": 286,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "green",
        "height": 3,
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "height": 50,
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": 3,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "height": 3,
        "left": 236,
        "position": "absolute",
        "top": 0,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "height": 50,
        "left": 283,
        "position": "absolute",
        "top": 0,
        "width": 3,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "height": 3,
        "left": 0,
        "position": "absolute",
        "top": 283,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "height": 50,
        "left": 0,
        "position": "absolute",
        "top": 236,
        "width": 3,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "height": 3,
        "left": 236,
        "position": "absolute",
        "top": 283,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "green",
        "height": 50,
        "left": 283,
        "position": "absolute",
        "top": 236,
        "width": 3,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.8)",
        "height": 186,
        "left": 0,
        "position": "absolute",
        "top": 50,
        "width": 3,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.8)",
        "height": 3,
        "left": 50,
        "position": "absolute",
        "top": 0,
        "width": 186,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.8)",
        "height": 186,
        "left": 283,
        "position": "absolute",
        "top": 50,
        "width": 3,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.8)",
        "height": 3,
        "left": 50,
        "position": "absolute",
        "top": 283,
        "width": 186,
      }
    }
  />
</View>
`;
